docs/api/guides/guide-to-creating-an-endpoint.md

```diff
@@ -139,21 +139,71 @@ For more information about service objects and when to create one: [Service Obje
 An important note about service objects: The service layer is where we will store our business logic and connect to the database. Once a service object is created, it will be passed in to the handler `NewPrimeAPIHandler` function in `pkg/handlers/primeapi/api.go`,
 and the handler will only be aware of the service object interface, while the service object will contain all of the rules and validations as well as accessing object from the database.
 
-Handlers must never hit the database. Ideally, endpoint handlers are for type validations.
+Handlers must never hit the database. Ideally, endpoint handlers are for type validations. 
 
-#### Steps to creating a new handler:
-1. Add a handler for the endpoint
-2. Add payload_to_model converters
-    * this is a good place to check data types and null values.
-3. model_to_payload functions
+Handlers are also where
+we convert from payload (Swagger types) to model (MilMove Go types). Generally, before calling the service
+object to process the request, you would call the appropriate `payload_to_model.go` function. And after returning
+from the the call to the service object you would call the appropriate `model_to_payload.go` function.
+
+### Example diagram of how Swagger calls our handler functions:
+* [MilMove Swagger call PDF](/files/swagger/MilMoveSwaggerCall.pdf)
+* [MilMove Swagger call PNG](/img/swagger/MilMoveSwaggerCall.png)
+
+### Steps to creating a new handler:
+1. **Add a handler for the endpoint.**
+   * Update the `api.go` file depending on which API you are updating:
+```
+pkg/handlers/
+	adminapi/api.go
```

Comment:
Adding these files is super helpful. Could we also add a brief explanation as to what these files are or what they do for steps 1-4? Unless its somewhere else in the file

Reply:
@kennethchow-truss  I also included a link to diagram to show how they work, did you see that?